breathecode/certificate/tests/urls/tests_certificate_cohort_id.py

```diff
@@ -665,7 +665,8 @@ def test_generate_certificate_good_request(self):
 
         url = reverse_lazy('certificate:certificate_cohort',
                            kwargs={'cohort_id': 1})
-        response = self.client.post(url, format='json')
+        data = {'layout_slug': 'vanilla'}
```

Comment:
change the file name from `tests_certificate_cohort_id.py` to `tests_cohort_id.py`

```python
    path('cohort/<int:cohort_id>',
         CertificateCohortView.as_view(),
         name='certificate_cohort'),
```